src/controllers/UsersController.php

```diff
@@ -148,6 +148,16 @@ public function postAction($request, $db){
                 $user_id = $user_mapper->createUser($user);
                 header("Location: " . $request->base . $request->path_info . '/' . $user_id, NULL, 201);
 
+                // autoverify for test platforms
+                if(isset($this->config['features']['allow_auto_verify_users']) 
+                    && $this->config['features']['allow_auto_verify_users']) {
+                        if($request->getParameter("auto_verify_user") == "true") {
```

Comment:
separate statements because I like feature toggles to be separate.  Which is another way of saying: no reason, that's just how my brain works.